build/wrapper.cc

```diff
@@ -200,10 +208,15 @@ std::shared_ptr<RayObject> get(const ObjectID object_id, int64_t timeout_ms) {
     std::vector<ObjectID> get_obj_ids = {object_id};
     auto status = worker.Get(get_obj_ids, timeout_ms, &objects);
     if (!status.ok()) {
-        return nullptr;
+        throw std::runtime_error(status.ToString());
+    }
+
+    if (objects.size() != 1) {
+        auto msg = "Size of RayObjects is " + std::to_string(objects.size());
+        auto st = Status::Invalid(msg);
+        throw std::runtime_error(st.ToString());
     }
 
-    RAY_CHECK(objects.size() == 1);
     return objects[0];
```

Comment:
Similarly to `CoreWorker::Put` we should have [`CoreWorker::Get`](https://github.com/ray-project/ray/blob/a03efd9931128d387649dd48b0e4864b43d3bfb4/src/ray/core_worker/core_worker.h#L665-L667) return a `Status` and pass back the object by reference.